src/app/Archives/AllArchivedRecordingsTable.tsx

```diff
@@ -301,6 +301,8 @@ export const AllArchivedRecordingsTable: React.FunctionComponent<AllArchivedReco
         onHeaderCheck={handleHeaderCheck}
         isLoading={isLoading}
         isEmpty={!recordings.length}
+        isEmptyFilterResult={false}
+        clearFilters={() => {/** TODO */}}
```

Comment:
Is there still something TODO here?

Reply:
Oh yeah, originally I was going to add the search filter for the All Archives View, then we decided to wait until Hareet merged his PR to prevent conflicts. It looks like his PR already has a search feature as of now - I can revert my changes on the `AllArchivedRecordingsTable`.